services/sddm-plymouth.service.in

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Simple Desktop Display Manager
+Documentation=man:sddm(1) man:sddm.conf(5)
+Conflicts=getty@tty1.service plymouth-quit.service
+Wants=plymouth-deactivate.service
+After=systemd-user-sessions.service getty@tty1.service plymouth-deactivate.service plymouth-quit.service
+
+[Service]
+ExecStart=CMAKE_INSTALL_FULL_BINDIR@/sddm
+Restart=always
+
+[Install]
+Alias=display-manager.service
```

Comment:
won't this conflict with our main service having the same alias?

Reply:
You have to choose which one you install